docs/overview.rst

```diff
@@ -1,380 +1,306 @@
-        Decimal('1')
-
-
-.. _overview_addresses:
-
-Addresses
-~~~~~~~~~~~~~~~~
-
-.. py:method:: Web3.isAddress(value)
-
-    Returns ``True`` if the value is one of the recognized address formats.
-
-    * Allows for both ``0x`` prefixed and non-prefixed values.
-    * If the address contains mixed upper and lower cased characters this function also
-      checks if the address checksum is valid according to `EIP55`_
-
-    .. code-block:: python
-
-        >>> Web3.isAddress('0xd3CdA913deB6f67967B99D67aCDFa1712C293601')
-        True
-
+Cryptographic Hashing
+---------------------
 
-.. py:method:: Web3.isChecksumAddress(value)
+- :meth:`Web3.keccak() <web3.Web3.keccak>`
+- :meth:`Web3.solidityKeccak() <web3.Web3.solidityKeccak>`
 
-    Returns ``True`` if the value is a valid `EIP55`_ checksummed address
 
+web3.eth API
+~~~~~~~~~~~~
 
-    .. code-block:: python
+Most of this library's functionality can be found under the ``web3.eth`` namespace.
+As a reminder, the :ref:`Examples <examples>` page will demonstrate how to use
+several of these methods.
 
-        >>> Web3.isChecksumAddress('0xd3CdA913deB6f67967B99D67aCDFa1712C293601')
-        True
-        >>> Web3.isChecksumAddress('0xd3cda913deb6f67967b99d67acdfa1712c293601')
-        False
 
+Fetching Data
+-------------
 
-.. py:method:: Web3.toChecksumAddress(value)
+Viewing account balances (:meth:`getBalance <web3.eth.Eth.getBalance>`), transactions
+(:meth:`getTransaction <web3.eth.Eth.getTransaction>`), and block data
+(:meth:`getBlock <web3.eth.Eth.getBlock>`) are some of the most common starting
+points in Web3.py.
 
-    Returns the given address with an `EIP55`_ checksum.
 
+API
+^^^
 
-    .. code-block:: python
+- :meth:`web3.eth.getBalance() <web3.eth.Eth.getBalance>`
+- :meth:`web3.eth.getBlock() <web3.eth.Eth.getBlock>`
+- :meth:`web3.eth.getBlockTransactionCount() <web3.eth.Eth.getBlockTransactionCount>`
+- :meth:`web3.eth.getCode() <web3.eth.Eth.getCode>`
+- :meth:`web3.eth.getProof() <web3.eth.Eth.getProof>`
+- :meth:`web3.eth.getStorageAt() <web3.eth.Eth.getStorageAt>`
+- :meth:`web3.eth.getTransaction() <web3.eth.Eth.getTransaction>`
+- :meth:`web3.eth.getTransactionByBlock() <web3.eth.Eth.getTransactionByBlock>`
+- :meth:`web3.eth.getTransactionCount() <web3.eth.Eth.getTransactionCount>`
+- :meth:`web3.eth.getUncleByBlock() <web3.eth.Eth.getUncleByBlock>`
+- :meth:`web3.eth.getUncleCount() <web3.eth.Eth.getUncleCount>`
 
-        >>> Web3.toChecksumAddress('0xd3cda913deb6f67967b99d67acdfa1712c293601')
-        '0xd3CdA913deB6f67967B99D67aCDFa1712C293601'
 
-.. _EIP55: https://github.com/ethereum/EIPs/issues/55
+Making Transactions
+-------------------
 
+The most common use cases will be satisfied with
+:meth:`sendTransaction <web3.eth.Eth.sendTransaction>` or the combination of
+:meth:`signTransaction <web3.eth.Eth.signTransaction>` and
+:meth:`sendRawTransaction <web3.eth.Eth.sendRawTransaction>`. If interacting
```

Comment:
The *"If interacting with smart contracts..."* part seems like it belongs on it's own line (maybe as a `.. note::`)